ledger/participant-integration-api/src/main/scala/platform/store/cache/MutableCacheBackedContractStore.scala

```diff
@@ -346,9 +348,17 @@ private[platform] object MutableCacheBackedContractStore {
   final class CacheUpdateSubscription(
       contractStore: MutableCacheBackedContractStore,
       subscribeToContractStateEvents: SubscribeToContractStateEvents,
+      metrics: Metrics,
       minBackoffStreamRestart: FiniteDuration = 100.millis,
   )(implicit materializer: Materializer)
       extends ResourceOwner[Unit] {
+    private val cacheUpdaterExecutionContext =
```

Comment:
similary here (ResourceOwner.forExecutorService, InstrumentedExecutor, mapAsync(1) comment)

or...do I understand it right, that CacheUpdateSubscription will be gone with the indexer bypass? In that case TODO comments might be enough (instead of solidifying)

Reply:
This will be gone. But I'll align it

Reply:
Will undo. Not necessary.